README.md

```diff
@@ -130,6 +160,8 @@ Default: latest semver tag
 
 #### writerOpts
 
+#### parserOpts
```

Comment:
No, `includeDetails` is `writerOpts`.

The point is if the arg is exact the same as `conventional-changelog`, don't list here (give a link instead). If there are changes or additions, describe the differences only. Maybe the docs should be improved but should be in a seperate PR.